data/json/requirements/tailoring.json

```diff
@@ -255,7 +255,7 @@
     "type": "requirement",
     "//": "Crafting Kevlar items, per 101 g of Kevlar; 10 g + excessive weight of material is wasted, producing Kevlar scraps as byproducts.  Time needed is usually 90 minutes per unit if hand-stitching.",
     "qualities": [ { "id": "SEW", "level": 2 }, { "id": "CUT", "level": 1 } ],
-    "components": [ [ [ "sheet_kevlar", 22 ] ], [ [ "filament", 1, "LIST" ] ] ]
+    "components": [ [ [ "sheet_kevlar", 2 ] ], [ [ "filament", 1, "LIST" ] ] ]
```

Comment:
This is not a bug, this is 110g, as per the comment.

Reply:
In that case, the comment should clarify that because I can bet the next contributor will make the same assumption... "5g * 22 = 110g" should be called out somewhere...

Reply:
Okay, so the problem is deeper than this. I think I tracked it down: the game seems to base the salvaged item count on each material item being 250 mL, and disregards the actual size. This can be found in iuse_actor.cpp, around line 1446 and down.

Reply:
Looks like another one needing fixing by #45779 or similar.

Reply:
Yep, #45779 puts it at 12 leather patches, 23 kevlar sheets. It's still not quite correct, but it's an order of magnitude more correct.

Reply:
Kevlar is about 1.5 g/mL or so, so the issue may be that a 5g sheet of kevlar should be about 3-4 mL, not 300. That's a very teeny tiny piece of kevlar so we probably want to change that to something like 30g, 80mL, and change the amount of kevlar in the crafting requirement to be 1 sheet.

Reply:
This is two kevlar sheets
![image](https://user-images.githubusercontent.com/42699974/103317306-e5fc8b80-49df-11eb-8000-e4ce7a634ccd.png)

Reply:
Right. Well, the issue is far beyond the scope I thought, so I'm just gonna close this pull request. I hope #45779 makes it into master soon!